java/src/com/ibm/streamsx/topology/TStream.java

```diff
@@ -963,27 +963,26 @@
      * </code>
      * </pre>
      * 
-     * Each parallel channel can be thought of as being assigned its own thread.
-     * As such, each parallelized stream function (filter and transform, in this
-     * case) are separate instances and operate independently from one another.
+     * Calling {@code parallel(3)} creates three parallel channels. Each of the channels contains 
+     * separate instantiations of the operators declared in the region.
```

Comment:
At this api level, operators are not really known. There are streams and functions.